app/controllers/chatrooms_controller.rb

```diff
@@ -5,13 +5,6 @@ def index
     @chatrooms = Chatroom.all
   end
 
-  def new
-    if request.referrer.split("/").last == "chatrooms"
-      flash[:notice] = nil
-    end
-    @chatroom = Chatroom.new
-  end
-
   def edit
     @chatroom = Chatroom.find_by(slug: params[:slug])
```

Comment:
Are we still allowing users to rename their chats or otherwise use the edit action?